### PR TITLE
Align naming in Door Lock cluster XML with the spec.

### PR DESCRIPTION
**Pull request by bzbarsky-apple**
REVIEW NOTE: Only the first commit contains manual changes. The rest is code-generated.

### CODE REVIEW

**Review by jmartinez-silabs — APPROVED** [review]

File: examples/all-clusters-app/all-clusters-common/all-clusters-app.zap [hunk 1]
```diff
@@ -6125,15 +6125,15 @@
           "enabled": 0,
           "commands": [
             {
-              "name": "LockDoor",
+              "name": "Lock Door",
```
Comment (jmeg-sfy) [inline examples/all-clusters-app/all-clusters-common/all-clusters-app.zap:6128]: :(

File: examples/all-clusters-app/all-clusters-common/all-clusters-app.zap [hunk 2]
```diff
@@ -12813,111 +12813,111 @@
...
               "code": 0,
               "mfgCode": null,
               "source": "client",
               "incoming": 1,
               "outgoing": 1
             },
             {
-              "name": "UnlockDoor",
+              "name": "Unlock Door",
```
Comment (jmeg-sfy) [inline examples/all-clusters-app/all-clusters-common/all-clusters-app.zap:12824]: :(

File: examples/all-clusters-app/all-clusters-common/all-clusters-app.zap [hunk 3]
```diff
@@ -12813,111 +12813,111 @@
...
               "code": 1,
               "mfgCode": null,
               "source": "client",
               "incoming": 1,
               "outgoing": 0
             },
             {
-              "name": "UnlockWithTimeout",
+              "name": "Unlock with Timeout",
```
Comment (jmeg-sfy) [inline examples/all-clusters-app/all-clusters-common/all-clusters-app.zap:12832]: i would favor to update the specs.... really

Comment (jmartinez-silabs) [reply examples/all-clusters-app/all-clusters-common/all-clusters-app.zap:12832]: +1

File: examples/all-clusters-app/all-clusters-common/all-clusters-app.zap (hunk 1 above)
Comment (bzbarsky-apple) [reply examples/all-clusters-app/all-clusters-common/all-clusters-app.zap:6128]: Matching the spec....  In practice, this does not affect the generated code, particularly.

File: examples/all-clusters-app/all-clusters-common/all-clusters-app.zap (hunk 2 above)
Comment (bzbarsky-apple) [reply examples/all-clusters-app/all-clusters-common/all-clusters-app.zap:12824]: Again, matching the spec.

File: examples/all-clusters-app/all-clusters-common/all-clusters-app.zap (hunk 3 above)
Comment (bzbarsky-apple) [reply examples/all-clusters-app/all-clusters-common/all-clusters-app.zap:12832]: I considered that, but it's a bit of a pain to do that in practice, compared with just changing the SDK.  I asked about this in the door lock channel; we'll see how that goes.

**Review by jmeg-sfy — APPROVED** [review]

File: examples/all-clusters-app/all-clusters-common/all-clusters-app.zap (hunk 3 above)
Comment (bzbarsky-apple) [reply examples/all-clusters-app/all-clusters-common/all-clusters-app.zap:12832]: OK, I filed https://github.com/CHIP-Specifications/connectedhomeip-spec/issues/6190

If someone has time to fix up the spec, I would really appreciate it.

Comment (jmeg-sfy) [reply examples/all-clusters-app/all-clusters-common/all-clusters-app.zap]: yep hopefully someone step forward from this group, thanks Boris for the issue filled

**Review by tcarmelveilleux — APPROVED** [review]

**Pull request merged**